.template/spec/support/serverspec.rb

```diff
@@ -7,7 +7,8 @@ module ServerSpecHelpers
   # Prebuild and run docker image before running the test
   # Because the docker api does not support docker compose
   def self.test_container
-    container_id = `docker ps -qf "name=#{ENV.fetch('APP_NAME')}_test"`
+    container_name = "#{ENV.fetch('APP_NAME')}-test"
```

Comment:
```suggestion
    container_name = "#{ENV.fetch('CONTAINERIZED_APP_NAME')}-test"
```

Reply:
As we don't have that environment variable inside the Docker image, therefore it will raise an error.

<img width="589" alt="image" src="https://github.com/nimblehq/rails-templates/assets/63148598/9c8f69f2-f958-4f28-a6ec-d7451122d20e">

I tried and I think we can keep using the current one.

Reply:
Oh, ok ok, well noted, thanks!